plugins/nf-co2footprint/src/main/nextflow/co2footprint/CO2FootprintConfig.groovy

```diff
@@ -59,16 +59,16 @@ class CO2FootprintConfig {
         file = config.file ?: CO2FootprintFactory.CO2FootprintTextFileObserver.DEF_FILE_NAME
         summaryFile = config.summaryFile ?: CO2FootprintFactory.CO2FootprintTextFileObserver.DEF_SUMMARY_FILE_NAME
         reportFile = config.reportFile ?: CO2FootprintFactory.CO2FootprintReportObserver.DEF_REPORT_FILE_NAME
-        def country
 
         ci = 475
         if (config.ci && config.country)
-            throw new IllegalArgumentException("Invalid combination of 'ci' and 'country' parameters specified for the CO2Footprint plugin. Please specify either 'ci' and 'country'!")
+            throw new IllegalArgumentException("Invalid combination of 'ci' and 'location' parameters specified for the CO2Footprint plugin. Please specify either 'ci' and 'location'!")
```

Comment:
Shouldn't the last part read "please specify either 'ci' OR 'location'"?

Reply:
Ups, yes of course